contracts/libraries/StoryOracleClient.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "./StoryOracleHelper.sol";
+import "../interfaces/IStoryOracleCallback.sol";
+
+
+abstract contract StoryOracleClient {
+    using StoryOracleHelper for StoryOracleHelper.Request;
+
+    uint256 private requestCount = 1;
```

Comment:
Wonder if requestCount here can be a Counter, what is the pros and cons of using Counter vs uint256?